Aptoide/aptoide-client-v8
# [MOB-230] task: remove similar apps bundle in app view experiment

**What does this PR do?**

   Remove similar apps bundle in app view experiment, returning to its original implementation.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Go to an app, check similar apps bundle order after clicking install. First Similar apps, second should be Similar APPC apps. In AppCoins app view, should be the reverse.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-230](https://aptoide.atlassian.net/browse/MOB-230)

**Code Review Checklist**
